ckan/lib/base.py

```diff
@@ -233,6 +231,44 @@ def set_pylons_response_headers(allow_cache):
     return render_template
 
 
+def _allow_caching(cache_force=None):
+    # Caching Logic
+
+    allow_cache = True
+    # Force cache or not if explicit.
+    if cache_force is not None:
```

Comment:
@Zharktas What can be the case when the `cache_force` will not be None? 

In the case of `pylons`, it is mention in the method `_pylons_prepare_renderer`

Reply:
@shubham-mahajan I guess the only case would be if it's passed explicitly from another function in the code. I guess it can be removed but it can be done in future refactorings.